papermill/cli.py

```diff
@@ -10,10 +10,18 @@
 
 from .execute import execute_notebook
 from .iorw import read_yaml_file
+from .api import Notebook as notebook
```

Comment:
So the intention is to print the papermill library version rather than the notebook version.

```from . import __version__ as papermill_version```

will get you the version of the library being used. What you have here will print `<property object at 0x7f54aa88baf8>` instead :)